Src/DependencyCollector/Shared/Implementation/FrameworkHttpProcessing.cs

```diff
@@ -49,6 +41,12 @@ public void OnBeginHttpCallback(long id, string resourceName)
             try
             {
                 DependencyCollectorEventSource.Log.BeginCallbackCalled(id, resourceName);
+                if (DependencyTableStore.Instance.IsDesktopHttpDiagnosticSourceActivated)
+                {
+                    // request is handled by Desktop DiagnosticSource Listener
+                    DependencyCollectorEventSource.Log.TrackingAnExistingTelemetryItemVerbose();
```

Comment:
I've already done that in the next PR #538, there was a special event added there